h/activity/query.py

```diff
@@ -18,9 +18,9 @@
 from h.search.query import (
     AuthorityFilter,
     TagsAggregation,
-    TopLevelAnnotationsFilter,
     UsersAggregation,
 )
+from h.search import TopLevelAnnotationsFilter
```

Comment:
The other `h.search.query` imports here will be changed to `h.search` imports in due time